project/Deps.scala

```diff
@@ -13,6 +13,11 @@ object Deps {
 
   def twitterUtil(mod: String) =
     "com.twitter" %% s"util-$mod" % "19.5.1"
+
+  def twitterCommonUtil = "com.twitter.common" % "util" % "0.0.121"
```

Comment:
I think the goal here is to stop depending on com.twitter.common altogether.  I think this means we also need to copy things like `com.twitter.common.base.Command` into our repo if they haven't already been copied into finagle-serversets